src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -20,7 +20,7 @@
 
 from .spikeinterfacerecordingdatachunkiterator import SpikeInterfaceRecordingDataChunkIterator
 from ..nwb_helpers import get_module, make_or_load_nwbfile
-from ...utils import dict_deep_update, FilePathType, calculate_regular_series_rate
+from ...utils import dict_deep_update, calculate_regular_series_rate, FilePathType
```

Comment:
```suggestion
from ...utils import dict_deep_update, FilePathType, calculate_regular_series_rate
```

Reply:
@bendichter if you like, we can setup `isort` in pre-commit and let it do the work for this kind of stuff

Reply:
> @bendichter if you like, we can setup `isort` in pre-commit and let it do the work for this kind of stuff

Yeah, that seems like a good idea